symbulate/distributions.py

```diff
@@ -413,18 +441,30 @@ class Gamma(Distribution):
 
     def __init__(self, shape, rate=1.0, scale=None):
         
-        if 0 < shape:
+        if shape is None:
```

Comment:
Remove this check. shape can never be None. It has no default value, so Python will require the user specify a value for shape.